worlds/sc2wol/docs/setup_en.md

```diff
@@ -1,62 +1,69 @@
-# Starcraft 2 Wings of Liberty Randomizer Setup Guide
+# StarCraft 2 Wings of Liberty Randomizer Setup Guide
+
+This guide contains instructions on how to install and troubleshoot the StarCraft 2 Archipelago client, as well as where
+to obtain a config file for StarCraft 2.
 
 ## Required Software
 
-- [Starcraft 2](https://starcraft2.com/en-us/)
-- [Starcraft 2 AP Client](https://github.com/ArchipelagoMW/Archipelago)
-- [Starcraft 2 AP Maps and Data](https://github.com/TheCondor07/Starcraft2ArchipelagoData)
+- [StarCraft 2](https://starcraft2.com/en-us/)
+- [The most recent Archipelago release](https://github.com/ArchipelagoMW/Archipelago/releases)
+- [StarCraft 2 AP Maps and Data](https://github.com/TheCondor07/Starcraft2ArchipelagoData)
 
-## General Concept
+## How do I install this randomizer?
 
-Starcraft 2 AP Client launches a custom version of Starcraft 2 running modified Wings of Liberty campaign maps
- to allow for randomization of the items
+1. Install StarCraft 2 and Archipelago using the first two links above. (The StarCraft 2 client for Archipelago is
+   included by default.)
+2. Click the third link above and follow the instructions there.
+3. Linux users should also follow the instructions found at the bottom of this page 
+   (["Running in Linux"](#running-in-linux)).
 
-## Installation Procedures
+## Where do I get a config file (aka "YAML") for this game?
```

Comment:
This would be better if it just covered how to set your randomizer settings. I don't believe there's much reason to mention "YAML" or explain why a config file is necessary. I think most people can understand that you would need to choose the settings for the level of randomization you experience.

Reply:
I mention "YAML" because of an incongruency between the generic Archipelago guides and the game-specific guides. All of the generic guides exclusively refer to "YAML"s, while many of the game guides refer to "config files". Personally, I think "config file" is better because it communicates the purpose of the thing, but I understand that "YAML" is catchy and quick to say/type. For as long as the two coexist, I think it's better to put the two names together in this document so newbies know they're referring to the same thing.

(Imo, the generic Archipelago guides are in need of much pruning/revisiting, and I intend to help with that and make this section completely unnecessary. Until that's done, though, I think it's useful and good to include this information for newcomers.)

Reply:
Yeah, yikes, that guide really does need a full rewrite. .gitblame me :)

Reply:
i think that "player settings" would be the better name than "config file"